stacker/blueprints/base.py

```diff
@@ -402,6 +405,36 @@ def render_template(self):
         version = hashlib.md5(rendered).hexdigest()[:8]
         return (version, rendered)
 
+    def parse_user_data(self, raw_userdata):
```

Comment:
i would be consistent about how you reference `user_data` vs `userdata`. i prefer `user_data`